### PR TITLE
i18n: update raidboss test timeline translations

### PR DESCRIPTION
**Pull request by MaikoTan**
@Jaehyuk-Lee can you review the korean part?

### CODE REVIEW

File: ui/raidboss/data/00-misc/test.ts [hunk 1]
```diff
@@ -514,6 +542,14 @@ const triggerSet: TriggerSet<Data> = {
...
         'Pentacle Sac': 'Pentacle Sac',
         'Engage': '시작',
+        'Two': '둘',
+        '(?<! )Three': '셋',
+        'Four': '넷',
+        'Six': '여섯',
+        'Ten': '열',
+        'Fifteen': '십오',
+        'Force Jump Three': '세로 뛰어넘기',
+        'Invisible': '보이지 않는',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/00-misc/test.ts:552]: Long time no see! 😄 

```suggestion
        'Two': '2',
        '(?<! )Three': '3',
        'Four': '4',
        'Six': '6',
        'Ten': '10',
        'Fifteen': '15',
        'Force Jump Three': '3으로 돌아가기',
        'Invisible': '타임라인 숨기기',
```

Using Arabic numerals is more natural. Interesting to recognize that Korean is the only one who uses number itself as a written text.

Comment (MaikoTan) [reply ui/raidboss/data/00-misc/test.ts]: That's fair.

And I have add another string that needs you to review again, sorry.

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/00-misc/test.ts]: `cactbot test combatant cast`

I don't understand what this means. Is this testing if recognizing skills which needs target works?

Comment (MaikoTan) [reply ui/raidboss/data/00-misc/test.ts]: This is a test for "whether cactbot could catch a spell casting game log". eg. Fire IV of Black Mage or something.

Comment (MaikoTan) [reply ui/raidboss/data/00-misc/test.ts]: By the way, I translate it in Japanese literally as "cactbot target cast test".

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/00-misc/test.ts]: I think it works in Korean just by transliterating that English one 🤣

File: ui/raidboss/data/00-misc/test.ts [hunk 2]
```diff
@@ -500,6 +529,7 @@ const triggerSet: TriggerSet<Data> = {
         'cactbot lang': 'cactbot 언어',
         'cactbot test response': 'cactbot 응답 테스트',
         'cactbot test watch': 'cactbot 탐지 테스트',
+        'cactbot test combatant cast': 'cactbot 대상 스킬 시전 테스트',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/00-misc/test.ts:532]:
```suggestion
        'cactbot test combatant cast': 'cactbot 타겟 스킬 시전 테스트',
```

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/00-misc/test.ts]: Ah, need one fix.
```
        'cactbot test combatant cast': 'cactbot 스킬 시전 테스트',
```

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/00-misc/test.ts]: @MaikoTan

Comment (MaikoTan) [reply ui/raidboss/data/00-misc/test.ts]: Done!

**Pull request merged**